website/content/en/docs/building-operators/golang/project_migration_guide.md

```diff
@@ -50,133 +66,185 @@ multigroup: true
 version: 2
 ...
 ```
-For multi-group project, the API Go type files are created under `apis/<group>/<version>` and the controllers would be created under `controllers/<group>`.
 
-## Create an API
+**Note**: For multi-group project, the API Go type files are created under `apis/<group>/<version>` and the controllers would be created under `controllers/<group>`.
+
+## Migrating APIs and Controllers
 
-Create a new API and its corresponding controller.
+Now, let's create the same API's that we have in our project with the legacy layout. In
+ case of memcached operator, a new API for kind `Memcached` and group/version  `cache/v1aplha1` is created with:
+
+```sh
+$ operator-sdk create api --group cache --version v1alpha1 --kind Memcached
+```
 
-In case of memcached operator, a new API for kind `Memcached` and group/version  `cache/v1aplha1` is created using the following command.
+Since in the [quick-start][quickstart]
+we created the  API and a Controller for the Memcached Kind then, we will press `[y]` for both cases which will result in `api/v1alpha1/memcached_types.go` and `controller/memcached_controller.go` created. 
 
-`operator-sdk create api --group cache --version v1alpha1 --kind Memcached`
+**Note** Ensure that you use the same values for the flags to recreate the same Go project with the same API's. If you have
+more than one API(CRD) you can add them via `operator-sdk create api` command. For further information check the [quick-start][quickstart]. 
 
-Press `y` when asked for creating resource and controller. This will scaffold the project and create the files `api/<version>/<kind>_types.go` and `controller/<kind>_types.go`.
+## Porting the code 
 
-For memcached operator project, `api/v1alpha1/memcached_types.go` and `controller/memcached_controller.go` are generated.
+The next step will be port our code to the new project. 
 
-### Migrate API type definitions
+### Migrating API's
 
-Copy over the API spec and status from the SDK project's `pkg/apis/<group>/<version>/<kind>_types.go` to Kubebuilder's `api/<version>/<kind>_types.go`.
+Now, let’s copy the API definition from `pkg/apis/<group>/<version>/<kind>_types.go` to `api/<version>/<kind>_types.go`. Note that is only required to copy the code from the Spec and Status fields.
 
-In our example, `MemcachedSpec` and `MemcachedStatus` from [`pkg/apis/cache/v1aplha1/memcached_types.go`][memcached_types] are copied to `api/v1alpha1/memcached_types.go`.
+**What to do about the Markers**
```

Comment:
Maybe hyperlinking **Marker** here to its definition is a good idea. Newbies can click and know right away.